File: 0075-sort-colors/0075-sort-colors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int i=0,j=0;
        for(i=0;i<nums.size();i++)
        {
            for(j=i;j<nums.size();j++)
            {
                if(nums[i]>nums[j])
                {
                    int temp=nums[i];
                    nums[i]=nums[j];
                    nums[j]=temp;
                }
            }
        }
    }
};

File: 0075-sort-colors/0075-sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        s1,s2,s0=0,0,0
        for i in range(len(nums)):
            if nums[i]==0: s0+=1
            elif nums[i]==1: s1+=1
            else: s2+=1

        for i in range(s0):
            nums[i]=0
        for i in range(s0,s0+s1):
            nums[i]=1
        for i in range(s0+s1,len(nums)):
            nums[i]=2
        